src/src/pages/capabilities/NewCapabilityDialog.js

```diff
@@ -63,7 +105,19 @@ export default function NewCapabilityDialog({
 
   const handleAddCapabilityClicked = () => {
     if (onAddCapabilityClicked) {
-      onAddCapabilityClicked(formData);
+      onAddCapabilityClicked({ ...formData, invitations: invitees });
+    }
+  };
+
+  const OnKeyEnter = (e) => {
+    if (e.key === "Enter") {
+      if (!Array.isArray(formData.invitations)) {
```

Comment:
reverse statements to reduce nesting

Reply:
Addressed in https://github.com/dfds/selfservice-portal/pull/133/commits/cb4b6bfc44c8d7dff3c4ea8e1c66e2dc416c35bd